site/messages/en.json

```diff
@@ -25,6 +25,7 @@
   "collection": "Collection {collectionId}",
   "connect-to-sync": "Please connect your wallet to search for approvals.",
   "connect-wallet": "Connect Wallet",
+  "connect-wallet-cta": "Connect your EVM Wallet",
```

Comment:
For consistency, we should have a single key and string representing the action.

```json
  "connect-wallet": "Connect your EVM Wallet",
```

Reply:
fixed